tool-base/src/main/java/io/spine/tools/java/fs/SourceFile.java

```diff
@@ -176,11 +207,10 @@ private static SourceFile forMessageOrInterface(DescriptorProto message,
             var result = forOuterClassOf(file);
             return result;
         }
-
     }
 
-    private static IllegalStateException invalidNestedDefinition(String filename,
-                                                                 String nestedDefinitionName) {
+    private static
+    IllegalStateException invalidNestedDefinition(String filename, String nestedDefinitionName) {
```

Comment:
@yevhenii-nadtochii this is a common formatting in Spine. Just for your information. 

@alexander-yevsyukov however, I would try to shorten both the method name and the name of the second argument. Within "new" formatting of these lines, it would probably look better.